app/robots/iCubGazeboV3/leftFingersHapticRetargetingParams.ini

```diff
@@ -161,3 +161,29 @@ no_states_kf            3
 no_measurement_kf       1
 q_matrix_kf             ( 10.0 150.0 100000.0 )
 r_matrix_kf             ( 0.0000001 )
+
+#####################
+## TACTILE SENSORS ##
+#####################
+## length of the tactile sensor port
+noTactileSensors        192
```

Comment:
Does the simulated robot have skin?

Reply:
I was testing it using some dumped skin data for the real robot, but we actually can remove it from this PR